cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -922,16 +922,16 @@ func nodeName(clusterName, role string, index int, uuid uuid.UUID) string {
 }
 
 func postCreate(ctx context.Context, clusterAccess *access.Adapter) error {
+	if !clusterWait {
```

Comment:
this feels a bit wrong, as if we do `--wait=false`, then it will never bootstrap a cluster at all (?)

Reply:
I agree that flags are confusing though

Reply:
maybe something like no bootstrap flag will work here.

Reply:
On other hand, if you don't want to wait for cluster to be ready you might not want to wait for bootstrap too, no?

Reply:
well... I agree it should be something like `--skip-bootstrap` but it's overall a mess :face_exhaling:

Reply:
Another thing I can suggest is `--bootstrap` flag which is true by default.

Reply:
But I don't like the fact that you have to remember that you should do both `--wait=false` and `--bootstrap=false`.
To me it looks logical to skip bootstrap if `--wait=false` :)

Reply:
`--wait=false` means you don't want to run health checks, e.g. they won't pass as I don't have CNI ready, but I still want to bootstrap.

`--with-init-node` is a legacy thing, but it allows to test init machine type, and also disables bootstrap (as you won't need it)

in Omni, we used both as we don't supply the machine config at all.

I would say we should have a better way to communicate these intentions, and rework the whole thing. 

What is the problem you're trying to fix with this PR? Should we work backwards from it?

Reply:
I want `--wait=false` not to demand `--with-init-node`.

Maybe we need another command that can just spin the nodes and that's it.
Going from that end we might need to extract machine and network creation bits into a separate chunk, then create a cmd that can do that.

Reply:
You can do that today, `pkg/provision` is a separate library, e.g. Sfyra uses it this way.